### PR TITLE
Update Readme for Popup and popup-content

### LINKED ISSUE #2120: Update Readme for Popup and popup-content
It seems that is possible to use it with react.

- [x] why 2 components? maybe hide it in storybook?
- [x] update readme
- [x] update "usage" plugin

### PR DESCRIPTION
**Pull request by marcwelz**
Fixes #2120

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

**Comment by markus-walther** [comment]
Marc, unfortunately other code violates ESlint rules, that's why CI tests failed on Azure:

`> npm run test-lint && prettier --check './src/**/*' && node ./scripts/build.test.js && npm run test-jest && npm run test-ui


> @axa-ch/pattern-library@2.0.0 test-lint /home/vsts/work/1/s
> eslint src/


/home/vsts/work/1/s/src/components/20-molecules/accordeon/index.js
  118:13  warning  Unexpected unnamed function  func-names

/home/vsts/work/1/s/src/components/30-organisms/modal/pw.ui.test.js
  6:22  warning  Unexpected unnamed function  func-names
`

File: src/components/20-molecules/popup/popup-button/README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
-# AXA Popup Button
+## AXA Popup Button
```
Comment (01231) [inline src/components/20-molecules/popup/popup-button/README.md]: Why this change?

File: src/components/20-molecules/popup/popup-content/README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
-# AXA Popup Content
+## AXA Popup Content
```
Comment (01231) [inline src/components/20-molecules/popup/popup-content/README.md]: Why this change?

**Review by 01231 — COMMENTED** [review]
In my opinion you could also delete the third readme in: `pattern-library/src/components/20-molecules/popup/README.md`

File: src/components/20-molecules/popup/popup-button/README.md (hunk 1 above)
Comment (marcwelz) [reply src/components/20-molecules/popup/popup-button/README.md]: Good question... 😁

**Review by 01231 — APPROVED** [review]
Nice!
